src/tree/help/HelpTreeDataProvider.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import TreeNode from '../TreeItem';
+import GetStartedNode from './GetStartedNode';
+import ReadDocumentationNode from './readDocumentationNode';
+import ReportIssueNode from './ReportIssueNode';
+import ReviewIssuesNode from './ReviewIssuesNode';
+
+export default class HelpTreeDataProvider implements vscode.TreeDataProvider<TreeNode> {
```

Comment:
Why can't we use `AzExtTreeDataProvider`?

Reply:
These are simple nodes with no dependency on Azure and simple to use.